tests/client_tests/stock_jdbc/src/test/java/stock_jdbc/JdbcMetaDataTest.java

```diff
@@ -222,11 +222,8 @@ public void test_getFunctionColumns() throws Exception {
     @Test
     public void test_getFunctions() throws Exception {
         try (var conn = DriverManager.getConnection(URL)) {
-            Assert.assertThrows(
-                "Unknown function: pg_get_function_result",
-                Exception.class,
-                () -> conn.getMetaData().getFunctions("", "", "s")
-            );
+            var results = conn.getMetaData().getFunctions("", "", "s");
+            assertThat(results.next(), is(false));
```

Comment:
Would it make sense to change the arguments to `getFunctions` so that we verify that some function is returned?